mycroft/listener/__init__.py

```diff
@@ -332,18 +332,35 @@ def create_hotword_engines(self):
         hot_words = self.config_core.get("hotwords", {})
         global_listen = self.config_core.get("confirm_listening")
         global_sounds = self.config_core.get("sounds", {})
-        for word in dict(hot_words):
+
+        main_ww = self.config_core.get("listener", {}).get("wake_word", "hey_mycroft").replace(" ", "_")
+        wakeupw = self.config_core.get("listener", {}).get("stand_up_word", "wake_up").replace(" ", "_")
+
+        for word, data in dict(hot_words).items():
             try:
-                data = hot_words[word]
+                # normalization step to avoid naming collisions
+                # TODO - move this to ovos_config package, on changes to the hotwords section this should be enforced directly
+                # this approach does not fully solve the issue, config merging may be messed up
+                word = word.replace(" ", "_")
```

Comment:
@NeonDaniel a bit out of scope but related, should we enforce this in every key for mycroft.conf, not only for hotwords?

Reply:
I agree its out of scope.. My main concern would be breaking compatibility with plugin config; maybe we could log warnings about it in `ovos_config` and then remove it when we release `1.0.0` there?